src/points/PointsHome.tsx

```diff
@@ -0,0 +1,156 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import { StackParamList } from 'src/navigator/types'
+import BackButton from 'src/components/BackButton'
+import React, { useEffect, useState, useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import { Screens } from 'src/navigator/Screens'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { Colors } from 'src/styles/colors'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import ActivityCardSection from 'src/points/ActivityCardSection'
+import BottomSheet, { BottomSheetRefType } from 'src/components/BottomSheet'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import { BottomSheetParams, PointsActivities } from 'src/points/types'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { PointsEvents } from 'src/analytics/Events'
+import CustomHeader from 'src/components/header/CustomHeader'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.PointsHome>
+
+export default function PointsHome({ route, navigation }: Props) {
+  const { t } = useTranslation()
+
+  // TODO: Use real points balance
+  const pointsBalance = 50
+
+  const bottomSheetRef = useRef<BottomSheetRefType>(null)
+
+  const [bottomSheetParams, setBottomSheetParams] = useState<BottomSheetParams | undefined>(
+    undefined
+  )
+  const onCardPress = (bottomSheetDetails: BottomSheetParams) => {
+    setBottomSheetParams(bottomSheetDetails)
+  }
+
+  useEffect(() => {
+    if (bottomSheetParams) {
+      bottomSheetRef.current?.snapToIndex(0)
+    }
+  }, [bottomSheetParams])
+
+  const onCtaPressWrapper = (onPress: () => void, activity: PointsActivities) => {
+    return () => {
+      ValoraAnalytics.track(PointsEvents.points_screen_card_cta_press, {
+        activity,
+      })
+      onPress()
+    }
+  }
+  return (
+    <SafeAreaView style={styles.outerContainer}>
```

Comment:
Fun little bug encountered during development for those knowledgeable/wanting to do some sleuthing - without the `outerContainer` style here, the page visually renders totally correctly, except is completely uninteractable (no scrolling, can't press on anything at all) under _only_ the following circumstances:
* Running on Android
* Page contents exceed screen size (meaning scrolling is required)
Notably, removing _either_ one of `CustomHeader` or `BottomSheet` made this issue go away... Satish and I spent some time trying to fix this, but still don't know why we were seeing this behavior :exploding_head:

Reply:
oh interesting.

Not directly related to that bug, but does the `ScrollView` extend to the bottom of the screen? If so the outer `SafeAreaView` should only apply to the top, and we should either add another `SafeAreaView` just for the bottom within the `ScrollView`, or adjust it's bottom padding using `useSafeAreaInsets()`.
So it doesn't hit an invisible wall on devices with a home bar.

This is how it should behave toward the bottom:
![scroll extend to bottom](https://github.com/valora-inc/wallet/assets/57791/6636d3e8-67e8-470b-a19f-482856e8aa63)

Reply:
having this safeareaview outside is fine since we've now got the custom header which shouldn't scroll :) to elaborate on Jean's comment, we should have `edges={['top']}` here and add a paddingBottom equivalent to the safe area bottom (or use another safe area view with the edges set to bottom only) on the scrollview contents to ensure that the end of the content isn't cut off.

weird android bug, i'm not sure i understand but perhaps you were experiencing [this one](https://valora-app.slack.com/archives/C024ZFDRFR9/p1710938221828669)?